locale/en/locale.cfg

```
@@ -121,79 +127,85 @@ ore_richness_normal=Normal
-upgrade_repair_message=Comfylatron: Train now gets repaired with additional repair kit at once.
-upgrade_repair_tooltip=Train uses more repair tools at once from Repair chest. Current: +__1__
+upgrade_repair_message=Comfylatron: The Chronotrain can now repair with an additional repair kit at once.
+upgrade_repair_tooltip=The Chronotrain uses more repair tools at once from the Repair Chest. Current: +__1__
 upgrade_water=Piping System
-upgrade_water_message=Comfylatron: Train now has piping system for additional water sources.
+upgrade_water_message=Comfylatron: The Chronotrain now has a piping system for additional water sources in each wagon. Am I allowed in the pool party??
 upgrade_water_tooltip=Add piping through wagon sides to create water sources for each wagon.
 upgrade_output=Output System
-upgrade_output_message=Comfylatron: Train now has output chests.
-upgrade_output_tooltip=Add comfylatron chests that output outside (into cargo wagon 2 and 3)
+upgrade_output_message=Comfylatron: The Chronotrain has been upgraded with output chests. Now it goes both ways!
+upgrade_output_tooltip=Adds output chests to wagons 2 and 3 which send items to the external wagon storage.
 upgrade_storage=Train Storage
-upgrade_storage_message=Comfylatron: Cargo wagons now have upgraded storage.
-upgrade_storage_tooltip=Add and upgrade storage chests to the sides of wagons.
+upgrade_storage_message=Comfylatron: Come look, the inside of the Chronotrain has upgraded storage.
+upgrade_storage_tooltip=Add or upgrade storage chests to line the edges of the insides of the Chronotrain.
 upgrade_poison=Poison Defense
-upgrade_poison_message=Comfylatron: I don't believe in your defense skills. I equipped train with poison defense.
-upgrade_poison_tooltip=Poison defense. Triggers automatically when train has low HP.\nMax charges : 4. Recharge timer for next use: __1__ min.
+upgrade_poison_message=Comfylatron: I don't believe in your defensive skills. I loaded another emergency poison defense into the locomotive...
+upgrade_poison_tooltip=Emergency poison defense. Triggers automatically when the Chronotrain has low HP.\nMax charges : 4. Recharge timer for next use: __1__ min.
 upgrade_fusion=Fusion Reactor
 upgrade_fusion_message=Comfylatron: One personal fusion reactor ready.
 upgrade_fusion_tooltip=Creates one Fusion Reactor.
 upgrade_mk2=Power Armor MK2
-upgrade_mk2_message=Comfylatron: I upgraded one armor to mk2.
+upgrade_mk2_message=Comfylatron: I upgraded an armor to MK2.
 upgrade_mk2_tooltip=Creates one Power Armor MK2
 upgrade_computer1=Comfylatron's Quest 1
-upgrade_computer1_message=Comfylatron: Thanks for fixing train navigation. I can now get us rid of very poor worlds. It will still need more work though, come back after jump.
-upgrade_computer1_tooltip=Progresses main quest.\nAll next worlds won't have "very poor" ore distribution.
+upgrade_computer1_tooltip=Progresses main quest.\nNew planets won't ever have "very poor" ore distribution.
```

```suggestion
upgrade_computer1_tooltip=Progresses main quest.\nNew destinations won't ever have "very poor" ore distribution.
```
we are getting rid of word planet, not adding it back again -.-'

---

Are we? Sounds good :P